datasource/timeouts/schema.go

```diff
@@ -48,15 +85,24 @@ func Attributes(ctx context.Context) schema.Attribute {
 	}
 }
 
-func attributesMap() map[string]schema.Attribute {
-	return map[string]schema.Attribute{
-		attributeNameRead: schema.StringAttribute{
-			Optional: true,
-			Validators: []validator.String{
-				validators.TimeDuration(),
-			},
+func attributesMap(opts Opts) map[string]schema.Attribute {
+	attribute := schema.StringAttribute{
+		Optional: true,
+		Description: `A string that can be [parsed as a duration](https://pkg.go.dev/time#ParseDuration) ` +
+			`consisting of decimal numbers, each with optional fraction and a unit suffix, such as ` +
```

Comment:
Nit: May be worth simplifying this further for common usage and to be less technical (since Terraform configuration language doesn't typically deal with other number types such as octal, etc):

```suggestion
			`consisting of numbers and unit suffixes, such as ` +
```